docs/add-ins/addin-tutorial.md

```diff
@@ -962,7 +1068,7 @@ Now that the UI is implemented, let's add the code behind it. Create a file name
         $('#not-configured').show();
       }
 
-      // When insert button is clicked, build the content
+      // When insert button is selected, build the content
```

Comment:
"Select Insert to build the content and insert into...."

Reply:
I've changed "clicked" to "selected" but left the phrasing the same -- since these are code comments describing what will happen when the button is selected (not instructions for the reader).